midend/convertErrors.h

```diff
@@ -16,98 +16,77 @@ limitations under the License.
 #ifndef _MIDEND_CONVERTERRORS_H_
 #define _MIDEND_CONVERTERRORS_H_
 
+#include <map>
```

Comment:
The toosl version of convertErrors was merged with the mid end version. They are basically equivalent, it is just that the mid end version did not have a cpp file.